tests/secrets/test_local_filesystem.py

```diff
@@ -226,9 +226,15 @@ def test_missing_file(self, mock_exists):
                schema: lschema
                login: Login
                password: None
-               port: 1234""",
+               port: 1234
+               extra_dejson:
```

Comment:
Can you add tests for connection with extra key?

Reply:
Added